### ISSUE TITLE
is this project still maintained?

### ISSUE THREAD

**Issue by orionrush** [hugomrdias/prettier-stylelint#34]
Hi - just checking on the health of the project.

**Comment by orionrush** [comment]
a respectful 'bump'?

**Comment by mindseyeblind** [comment]
@orionrush seems dead, try https://github.com/prettier/stylelint-prettier

**Comment by orionrush** [comment]
yes, but this isn't ideal, as stylelint-prettier inverts the workflow. I want to prettify and then apply linting rules. I've reverted to just doing this in sequence manually.

**Comment by SharakPL** [comment]
@orionrush did you find any solution for that? another package maybe?

E: [This one](https://stackoverflow.com/a/54757106/6389945) seems to be working just fine :)

**Comment by orionrush** [comment]
Take a look at [stylelint-prettier](https://github.com/prettier/stylelint-prettier), it’s not as ideal of a workflow, but it’s actively maintained.

**Comment by orionrush** [comment]
@SharakPL 
Yes, your SO link is about [stylelint-prettier](https://github.com/prettier/stylelint-prettier), and those rules will help. 

However, I liked the workflow of this package (prettier-stylelint) because passing the document through prettier first and then enforcing stylelint rules makes more sense. Prettier has less customization - it doesn't care™️, and so, if we pass the document through stylelint firsts and _then_ Prettier, Prettier can undo some of the rules that you want to enforce in stylelint.

**Comment by ntwb** [comment]
It looks like https://github.com/prettier/stylelint-prettier now uses
> https://github.com/prettier/stylelint-config-prettier

This was previously at the following URL but now part of the Prettier project:
> https://github.com/shannonmoeller/stylelint-config-prettier

It would be great to see this package deprecated and link to the above _shared config_

**Comment by SharakPL** [comment]
> 
> It would be great to see this package deprecated and link to the above _shared config_

Actually it wouldn't since [prettier team recommends installing prettier-stylelint](https://github.com/prettier/prettier-vscode/issues/1051#issuecomment-553502775).  Although I can't make it work anyway 😞

**Comment by zmeyc** [comment]
There's a similar plugin `effective-prettier-vscode` which runs prettier, eslint and stylelint in order. I had to switch to it because prettier-stylelint stopped working.

https://marketplace.visualstudio.com/items?itemName=sebastian-software.effective-prettier-vscode
https://github.com/sebastian-software/effective-prettier
https://github.com/sebastian-software/effective-prettier-vscode

`yarn add --dev @effective/prettier`, install VSCode plugin, in .code-workspace: `"editor.defaultFormatter": "sebastian-software.effective-prettier-vscode"`
